libs/types/src/cdf/ballot-definition/convert.ts

```diff
@@ -45,6 +58,174 @@ export function convertVxfElectionToCdfBallotDefinition(
   const stateId = vxfElection.state.toLowerCase().replaceAll(' ', '-');
   const electionDate = dateString(new Date(vxfElection.date));
 
+  const precinctSplits: Record<
```

Comment:
I think at some point it would be nice to integrate our split precinct support into the VXF format. Currently the flow looks like this:
- User enters explicit split precinct data which in VxDesign
- That gets converted to VXF (encoding splits in ballot styles)
- The VXF is then converted to CDF for export (back to explicit split precincts)
- The CDF is converted back to VXF on import to VxAdmin (back to encoding splits in ballot styles)

It's a bit convoluted, but I think it works for now.